onnxmltools/convert/lightgbm/operator_converters/LightGbm.py

```diff
@@ -10,7 +10,7 @@
 from collections import Counter
 from lightgbm import LGBMClassifier, LGBMRegressor
 from ...common._registration import register_converter
-from .TreeEnsemble import _get_default_tree_classifier_attribute_pairs
+from ...sklearn.operator_converters.TreeEnsemble import _get_default_tree_classifier_attribute_pairs
```

Comment:
move it in the common folder to avoid cross reference among converters.
And remove _ from the beginning of the function name.